airflow/utils/log/file_task_handler.py

```diff
@@ -78,8 +79,6 @@ def _set_task_deferred_context_var():
 
 
 def _fetch_logs_from_service(url, log_relative_path):
-    import httpx
-
```

Comment:
Why was this import moved to top of the file? is it due to pre commit change?

Reply:
No, I did this because I need this module on line 506 (httpx.UnsupportedProtocol). I looked at the blame and there didn't seem to be any reason for this to be function scoped so moving it to module scoped.

Reply:
Actually, it looks like Ash moves this out of module scope to function for perf optimization here: https://github.com/apache/airflow/commit/1a8a897120762692ca98ac5ce4da881678c073aa#diff-e7f34f73940eb52d92bb991abedc1c963431c5373c12dff739c8fb7d03e93d3aL24

Going to move it back to function scope in a follow up PR

Reply:
PR: https://github.com/apache/airflow/pull/36753